components/Banner/BannerWithImage.js

```diff
@@ -0,0 +1,91 @@
+import React from 'react'
+import styled from 'styled-components'
+
+import { propTypes, defaultProps } from './propTypes'
+import { rem, headerFont } from '../../styling/typography'
+import {
+  extraExtraSmallSpacing,
+  smallSpacing,
+  mediumSpacing,
+  extraLargeSpacing,
+} from '../../styling/sizes'
+import Link from '../Link'
+import Button from '../Button'
+import Heading from './Headline'
+import { background } from './styles'
+import { lgBreakpoint, mdBreakpoint } from '../../styling/breakpoints'
+
+const Container = styled.div`
+  margin-right: 0;
+  margin-left: 0;
+`
+
+const Image = styled.div`
+  background: url(${({ image }) => image}) center no-repeat;
+  background-size: cover;
+  min-height: 300px;
+`
+
+const ContentContainer = styled.span`
+  ${background};
+  padding: ${smallSpacing};
+  color: white;
+  
+  @media(min-width: ${mdBreakpoint}) {
+    padding: ${mediumSpacing};
+  }
+  
+  @media(min-width: ${lgBreakpoint}) {
+    padding: ${mediumSpacing} ${extraLargeSpacing};
+  }
+`
+
+const SubHeading = styled.span`
+  display: block;
+  font-size: ${rem('24px')};
+  font-family: ${headerFont};
+  font-weight: 500;
```

Comment:
I feel like a parrot but shouldn't we be using shared styles for all our headings?

Reply:
I thought it is more of a unique style here. We can make a task on trello to make a shared header styles folder. It would be probably too much for this PR.